pkg/apimachinery/patch/patch_test.go

```diff
@@ -75,4 +76,48 @@ var _ = Describe("PatchSet", func() {
 		patches := patch.New()
 		Expect(patches.IsEmpty()).To(BeTrue())
 	})
+
+	DescribeTable("should add the patch bytes", func(p string, expectErr bool) {
+		err := patch.New().AddPatch([]byte(p))
+		if expectErr {
```

Comment:
This is a hint that a test table is not a fit.
There is no point in grouping the two scenarios if you use a boolean to differentiate them inside the common part.

I would recommend to create two independent tests instead.